### PR TITLE
25 navbar rebase

### PR DESCRIPTION
**Pull request by djstein**
Add in new navbar for site wide applications. Also includes large buttons to rest of labsite applications on root page.

### CODE REVIEW

File: templates/labsite/home.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<!-- Add padding to the top -->
+
+<div class="col-md-100" style="margin-top:35px">
+
+<div class="row">
+ {% for apps in labsite_apps %}
+      <div class="col-md-4">
+        <a
+            class='btn btn-default btn-block well'
+            style='hover: grey'
```
Comment (rpkilby) [inline templates/labsite/home.html]: Pull the inline style out into a class in the site style sheet.

File: templates/labsite/home.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                class="fa {{ apps.font_image }}"
+                style="font-size:15em; 
+                {% if apps.margin > 0 %}
+                margin-right: {{apps.margin}}px;
+                {% endif %}
+                color: {{ apps.color }};">
+            </i>
+            </br>
+            <p
+                style="font-size:25px; color: white boarder: black">
```
Comment (rpkilby) [inline templates/labsite/home.html]: same here

File: templates/labsite/home.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{% load staticfiles %}
+
+{% block content %}
+
+<!-- Add padding to the top -->
+
+<div class="col-md-100" style="margin-top:35px">
```
Comment (rpkilby) [inline templates/labsite/home.html]: I don't think 'col-md-100' does anything.

File: templates/labsite/home.html (hunk 1 above)
Comment (wbrefvem) [inline templates/labsite/home.html]: @djstein I think @rpkilby just said you have style but no class. You gonna take that??

**Pull request merged**